nbgrader/docs/source/user_guide/advanced.rst

```diff
@@ -11,71 +11,6 @@ Running nbgrader with JupyterHub
 
 Please see :doc:`/configuration/jupyterhub_config`.
 
-.. _assignment-list-installation:
```

Comment:
So the *enabling* of extensions also works systemwide for all users? If so, I think that should be made more explicit in the main install page (currently it just says you can *install* the extensions systemwide, but it doesn't make a mention of enabling systemwide). In that case, I think maybe we can move the stuff I flagged below into the main install instructions.

If not, then I think we should definitely keep this section.